README.md

```diff
@@ -43,3 +43,22 @@ git config --global core.eol lf
 git rm --cached -r .
 git reset --hard
 ```
+### Publish
+
+To publish the extension to marketplace you must have been added as a contributor in the publisher. If you have been added, then follow the next steps:
+
+1. First, you should generate a personal access token in Azure DevOps that you can do in the next URL https://dev.azure.com/celerik/_usersSettings/tokens

```

Comment:
Please, you specify the necessary permissions to allow deploy process (Marketplace manage) and if you can attach image with that, better